__tests__/cleanFile.test.js

```diff
@@ -35,4 +34,14 @@ describe('cleanFile', () => {
     expect(helpers.fileDoesNotExist('./__tests__/__output/test.txt')).toBeTruthy();
   });
 
+  describe('if a file does not exist', () => {
+    it('should not throw', () => {
+      try {
+        cleanFile('test.txt', format, {buildPath: '__tests__/__output/'}, {});
```

Comment:
Looks good, let’s rename to nonexistent.txt for clarity?

Reply:
Could we also change this to use an `expect` method like we do in other tests checking if something should or shouldn't throw? For example:
https://github.com/amzn/style-dictionary/blob/fa6ebeb6d6120fda4ca04934b554758a3727988f/__tests__/exportPlatform.test.js#L42